python/ray/serve/tests/test_grpc.py

```diff
@@ -576,5 +576,63 @@ async def Streaming(self, *args):
         r.result()
 
 
```

Comment:
Could you add a unit test that checks whether `RayServegRPCContext` is serializable?